src/braket/circuits/noises.py

```diff
@@ -1320,11 +1330,7 @@ def kraus(
         )
 
     def to_dict(self) -> dict:
-        return {
-            "__class__": self.__class__.__name__,
-            "matrices": self._matrices,
```

Comment:
Could we serialize this as `self._matrices.tolist()` ?

Reply:
Yes, I was hoping to avoid it in this PR, because from the doc:
"The array may be recreated via a = np.array(a.tolist()), although this may sometimes lose precision."

Reply:
Do you think that would be OK?

Reply:
Also, it contains "complex" objects, which are not serializable.

Reply:
Oh, ok. In that case we can leave it unimplemented for now.